feathr-impl/src/main/scala/com/linkedin/feathr/offline/job/FeatureTransformation.scala

```diff
@@ -888,7 +888,6 @@ private[offline] object FeatureTransformation {
     val features = transformers map {
       case extractor: AnchorExtractor[IndexedRecord] =>
         val features = extractor.getFeatures(record)
-        print(features)
```

Comment:
There is a `print(features)` at line 1426 as well

Reply:
Removed.